modular_skyrat/master_files/code/modules/mob/living/carbon/human.dm

```diff
@@ -0,0 +1,12 @@
+/mob/living/carbon/human/ZImpactDamage(turf/T, levels)
+	//Non cat-people smash into the ground
+	if(!isfelinid(src))
```

Comment:
```suggestion
	if(!isfelinid(src) || !istajaran(src) )
```
When https://github.com/Skyrat-SS13/Skyrat-tg/pull/7844 merged

Reply:
I really don't think it should effect Tajaran, as they don't have any of the other special Felinid things.

Reply:
Then give them the felinid things dangit